topics/releases.md

```diff
@@ -1,144 +1,111 @@
+For this reason, it is not recommended to run a Valkey cluster with servers running on different minor versions.
+Users should avoid new features until all servers in the cluster have been upgraded.
 
-### Minor versions
+Commands may also be marked as **deprecated** in minor versions.
+Deprecated commands are not removed, instead a replacement command or an alternative to using the command will be defined in the same minor version.
 
-Minor versions usually deliver maturity and extended functionality.
+### Major versions
 
-Upgrading between minor versions does not introduce any application-level
-compatibility issues.
+*Major* versions are released with significant functionality that may break backwards compatibility or alter key performance characteristics.
+Examples of significant functionality includes altering the behavior of an existing command, removing previously deprecated commands, changing the default value of configs, and significant refactoring for performance improvements.
 
-Minor releases may include new commands and data types that introduce
-operations-related incompatibilities, including changes in data persistence
-format and replication protocol.
+Upgrading from a previous major version is intended to be safe, but should be approached with caution.
+You should carefully read the release notes before performing a major version upgrade.
+Although Major versions can introduce breaking changes, data replicated from primaries to replicas will always be sent in a backward compatible format.
+You should always upgrade replicas before upgrading primaries in order to ensure data consistency.
 
-### Major versions
+The Valkey community strives to make as few backwards breaking changes as possible.
+When breaking changes are required, we will also strive to provide a way to mitigate the impact without incurring downtime to your application.
 
-Major versions introduce new capabilities and significant changes.
+## Release schedule
 
-Ideally, these don't introduce application-level compatibility issues.
+The Valkey community strives to release a stable major version once a year.
+Stable minor versions are created as needed in between major releases, and we aim to release at least one minor version a year.
 
-## Release schedule
+### Release candidate
 
-A new major version is planned for release once a year.
+New minor and major versions of Valkey begin by branching off the `unstable` branch as an initial release candidate branch with a name that takes the form of `*major.minor*`, example `7.2`.
+The first release candidate, or rc1, is released once it can be used for development purposes and for testing the new version.
+Release candidate versions will start with a patch version of "0" and will take the form `*major.minor.patch-rcN*`, example `7.2.0-rc1` followed by `7.2.0-rc2`.
```

Comment:
`` `*major.minor*` `` is rendered as `*major.minor*` with the stars visible. I think you may want to put the backticks inside the stars?

`` *`major.minor`* `` rendered as *`major.minor`*.

Another option is to just avoid backticks here. A version number isn't really code. We could combine bold and italics like

_major_**.**_minor_**.**_patch_**-rc**_N_ (where bold is fixed content and italics is variable content)

```suggestion
New minor and major versions of Valkey begin by branching off the `unstable` branch as an initial release candidate branch with a name that takes the form of *`major.minor`*, example `7.2`.
The first release candidate, or rc1, is released once it can be used for development purposes and for testing the new version.
Release candidate versions will start with a patch version of "0" and will take the form *`major.minor.patch-rcN`*, example `7.2.0-rc1` followed by `7.2.0-rc2`.
```